tests/green_mode/green_modeTest.py

```diff
@@ -356,5 +373,143 @@ def test_bear_test_fun_2(self):
+        printer = ConsolePrinter()
+        input_ = [{TestLocalBear: [{'filename': 'a.py', 'some_setting': 3},
+                                   {'filename': 'b.py', 'some_setting': 3},
+                                   {'filename': 'c.py', 'some_setting': 4},
+                                   {'filename': 'd.py',
+                                    'some_other_setting': 'x'}]},
+                  None]
+        data_struct = generate_data_struct_for_sections(input_)
+        test_data_struct = {TestLocalBear: [[{'filename': ['a.py', 'b.py'],
+                                              'some_setting': 3},
+                                             {'filename': ['c.py'],
+                                              'some_setting': 4}],
+                                            [{'some_other_setting': 'x',
+                                              'filename': ['d.py']}]]}
+        self.assertEqual(data_struct, test_data_struct)
+        project_files = ['a.py', 'b.py', 'c.py', 'd.py']
+        coafile = '.coafile.green'
+        full_path = str(Path(__file__).parent.parent.parent)
+        full_path_coafile = str(Path(__file__).parent.parent.parent / coafile)
+        with patch('os.walk') as mockwalk:
+            mockwalk.return_val = mockwalk.return_value = [
+                ('', (), ('a.py', 'b.py', 'c.py', 'd.py')), ]
+            generate_green_mode_sections(data_struct, full_path,
+                                         project_files, ['x'],
+                                         printer)
+        contents = ""
+
+        with open(full_path_coafile) as f:
+            for line in f.readlines():
+                contents += line
+        full_path_glob = full_path + os.sep + '**'
+        test_contents = ''\
+                        '[all]\n'\
+                        'ignore = x\n'\
+                        '[all.TestLocalBear1]\n'\
+                        'ignore += a.py, b.py, c.py, d.py\n'\
+                        'bears = TestLocalBear\n'\
+                        'some_setting = 3\n'\
+                        'files = a.py, b.py, ' + full_path_glob + '\n'\
+                        '[all.TestLocalBear2]\n'\
+                        'ignore += a.py, b.py, c.py, d.py\n'\
+                        'files = c.py, ' + full_path_glob + '\n'\
+                        'bears = TestLocalBear\n'\
+                        'some_setting = 4\n'\
+                        '[all.TestLocalBear3]\n'\
+                        'ignore += a.py, b.py, c.py, d.py\n'\
+                        'files = d.py, ' + full_path_glob + '\n'\
+                        'bears = TestLocalBear\n'\
+                        'some_other_setting = x\n'
+        # Since the order of settings within a seciton is volatile.
+        for line in test_contents.split('\n'):
+            if line == 'ignore = x':
+                continue  # Since the path depends on the test directory
+            self.assertIn(line, contents)
```

Comment:
something wrong with `full_path_glob`

windows output is  `C:projectscoala-quickstart**` instead of expected `C:\\projects\\coala-quickstart\\**`